lib/airbrake-ruby/backtrace.rb

```diff
@@ -188,6 +193,21 @@ def match_frame(regexp, stackframe)
 
         Patterns::GENERIC.match(stackframe)
       end
+
+      def read_code_hunk(logger, frame)
+        code_hunk = Airbrake::CodeHunk.new(frame[:file], frame[:line]).to_h
+        return unless code_hunk
+
+        unless code_hunk.key?(0)
+          frame[:code_hunk] = code_hunk
```

Comment:
Let's rename to just `code` for brevity? Anyway in JSON it should be `codeHunk`, not `code_hunk`

Reply:
👍